tests/bits/step-16.cc

```diff
@@ -1,6 +1,6 @@
 // ---------------------------------------------------------------------
 //
-// Copyright (C) 2005 - 2013 by the deal.II authors
+// Copyright (C) 2005 - 2013, 2015 by the deal.II authors
```

Comment:
same here with the copyright line.

I thought we have agreed on keeping it as simple as possible, i.e. first [year of introduction] - [last year of modification]. If not, I have already changed around 1000 file headers :-]